### PR TITLE
updated terrafrom template - added readme.md

### PR DESCRIPTION
**Pull request by ovt12**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules
+.terraform
+*.tfstate.*
```
Comment (Charlie-robin) [inline .gitignore]: Hey remove the dots ->  `*tfstate*` 
The state file hasn't been ignored.

File: playbooks/inventory.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[node]
+54.237.64.167 ansible_ssh_private_key_file=~/.ssh/groupone-awskey.pem
```
Comment (Charlie-robin) [inline playbooks/inventory.txt:2]: Can you make sure everyone has the key?

File: terraform.tfstate [hunk 3]
```diff
@@ -0,0 +1,478 @@
+{
```
Comment (Charlie-robin) [inline terraform.tfstate]: Delete this file

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
Hey, a couple of changes to work on.

File: playbooks/inventory.txt (hunk 2 above)
Comment (ovt12) [reply playbooks/inventory.txt:2]: I've notified everyone in the group to add the key into their ssh file

**Review by Charlie-robin — APPROVED** [review]

**Pull request merged**